src/ngraph/frontend/onnx_import/utils/common.cpp

```diff
@@ -46,6 +46,39 @@ namespace ngraph
                                        static_cast<onnx::TensorProto_DataType>(onnx_type)));
             }
 
+            std::size_t convert_negative_axis(std::int64_t axis, std::size_t tensor_rank)
+            {
+                if (axis >= 0)
+                {
+                    return std::min(static_cast<size_t>(axis), tensor_rank);
```

Comment:
I wonder if we should adjust too big axis or just throw an exception.
@postrational what do you think about it?